### PR TITLE
feat(country-map): added new countries in country-chart-map

### PR DESCRIPTION
**Pull request by Hammad-Raza**
generated geojson file via jupyter and added new countries

### CODE REVIEW

**Review by zhaoyongjie — COMMENTED** [review]
~~Thanks for the contribution! quick first pass comments.~~

File: superset-frontend/plugins/legacy-plugin-chart-country-map/src/countries.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@
  * under the License.
  */
 
+/* eslint-disable import/no-webpack-loader-syntax, import/no-unresolved */
```
Comment (zhaoyongjie) [inline superset-frontend/plugins/legacy-plugin-chart-country-map/src/countries.ts:20]: Why we need to add these rules here?

**Review by zhaoyongjie — COMMENTED** [review]
Thanks for the contribution! quick first pass comments.

File: superset-frontend/plugins/legacy-plugin-chart-country-map/src/countries.ts (hunk 1 above)
Comment (Hammad-Raza) [reply superset-frontend/plugins/legacy-plugin-chart-country-map/src/countries.ts]: typo, my bad

**Comment by zhaoyongjie** [comment]
@Hammad-Raza Could you run `npm run lint-fix` to fix CI in the `superset-frontend` folder? Thanks!

**Review by zhaoyongjie — APPROVED** [review]
LGTM! Thanks for your contribution. one no-blocking suggestion improves welcome!

replace snakeCase with space splited words in countries selection list
image

**Pull request merged**